indexer/packages/postgres/src/stores/subaccount-usernames-table.ts

```diff
@@ -97,20 +97,23 @@ export async function findByUsername(
   return (await baseQuery).find((subaccountUsername) => subaccountUsername.username === username);
 }
 
-export async function getSubaccountsWithoutUsernames(
+export async function getSubaccountZerosWithoutUsernames(
+  limit: number,
   options: Options = DEFAULT_POSTGRES_OPTIONS):
   Promise<SubaccountsWithoutUsernamesResult[]> {
   const queryString: string = `
-    SELECT id as "subaccountId"
+    SELECT id as "subaccountId", address
```

Comment:
nit: Does ordering matter? I see that where this function used, if there is a collision there is a retry with a new nonce. If a list of subaccounts with collisions is traversed in a different order when re-generating usernames then there will be differences for the subaccounts with collisions right?

Reply:
Good callout, adding `ORDER BY`. It's still technically possible for two different indexer instances of the same network to result in different generation, since it can be affected by timing at which the task runs vs. the subaccount it has indexed.